.github/workflows/api_update_reminder.yml

```diff
@@ -0,0 +1,16 @@
+on:
+  - workflow_dispatch # to allow for manual testing
+  - schedule:
+      cron: "0 0 1 3,6,9,12" # At 00:00 on 1st of March, June, September, and December
```

Comment:
I wonder if a month lead time might be a little too long. Should we change this to trigger on the 15th of those months?

Reply:
I padded as I thought testing for the new release may uncover things that need to be updated/fixed.

Reply:
Makes sense, my main concern was whether there was a risk of things being added during this period, after we make our changes.